code/src/pages/products/ProductsPage.tsx

```diff
@@ -1,7 +1,48 @@
+import AppTypography from "@/components/app-typography/AppTypography";
+import AppBox from "@/components/app-box/AppBox";
+import ProductCard from "@/components/product-card/ProductCard";
 import PageWrapper from "@/layouts/page-wrapper/PageWrapper";
+import { useGetProductsQuery } from "@/store/api/productsApi";
+
+import "./ProductsPage.scss";
 
 const ProductsPage = () => {
-  return <PageWrapper>Products</PageWrapper>;
+  const { data: products, isLoading } = useGetProductsQuery();
+
+  //@TODO Create Skeleton for product items
+  if (isLoading) return <AppTypography>Loading...</AppTypography>;
```

Comment:
Did you think to create HOC `withLoading`? Maybe also add it as TODO.

Reply:
> Did you think to create HOC `withLoading`? Maybe also add it as TODO.

We discussed maybe create `Skeleton` component for product item view with loading.
That would be great